imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -1,8 +1,35 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import moment from "moment";
+import { Reaction } from "/client/api";
 import { Badge, ClickToCopy } from "@reactioncommerce/reaction-ui";
 
+/**
+ * getBillingInfo
+ *
```

Comment:
It's not quite canonized yet, but private methods/functions should use the `@private` tag in the jsdoc so they don't get rendered out to our docs.